superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py

```diff
@@ -43,14 +43,13 @@ def add_parser_arguments(self):
         self._parser.add_argument(
             '--operations',
             type=str,
-            nargs='+',
-            default=list(self.__operations.keys()),
+            default='allreduce',
             help='Nccl operations to benchmark, e.g., {}.'.format(' '.join(list(self.__operations.keys()))),
         )
         self._parser.add_argument(
             '--ngpus',
             type=int,
-            default=8,
+            default=1,
```

Comment:
Does 1 gpu work?

Reply:
it's for mpi mode, each process use 1 gpu, but if you only run 1 gpu in total, it will not raise error, just got 0 busbw.

Reply:
Got it, thanks.

Reply:
1 gpu per process is the default setting in nccl-tests; 8 gpu will get error on 4-GPU node